packs/tfe_fdo_nomad/variables.hcl

```diff
@@ -0,0 +1,228 @@
+  default     = "tfe"
+}
+
+variable "tfe_database_parameters" {
+  description = "The database parameters to be used by TFE while connecting to the database."
+  type        = string
+  default     = "sslmode=require"
+}
+
+variable "tfe_object_storage_type" {
+  description = "The object storage type to be used by TFE"
+  type        = string
+  default     = "s3"
+}
+
+variable "tfe_object_storage_s3_bucket" {
+  description = "The name of the object storage bucket to be used by TFE"
+  type        = string
+  default     = "tfe"
+}
+
+variable "tfe_object_storage_s3_region" {
+  description = "The region of the object storage bucket used by TFE"
+  type        = string
+  default     = "us-west-2"
+}
+
+variable "tfe_object_storage_s3_use_instance_profile" {
+  description = "The instance profile setting for accessing the object storage bucket to be used by TFE"
+  type        = string
+  default     = "false"
+}
+
+variable "tfe_object_storage_s3_endpoint" {
+  description = "The ednpoint of the S3 compatible object storage to be used by TFE"
+  type        = string
+  default     = ""
+}
+
+variable "tfe_object_storage_s3_access_key_id" {
+  description = "The secret key id of the S3 compatible object storage to be used by TFE"
+  type        = string
+}
+
+variable "tfe_redis_host" {
+  description = "The name of the redis host to be used by TFE"
+  type        = string
+}
+
+variable "tfe_redis_user" {
+  description = "The user to be assumed by TFE to connect to redis."
+  type        = string
+  default     = ""
+}
+
+variable "tfe_redis_use_tls" {
+  description = "The tls settings for redis to be used by TFE"
+  type        = string
+  default     = "false"
+}
+
+variable "tfe_redis_use_auth" {
+  description = "The auth settings to be used by redis"
+  type        = string
+  default     = "false"
+}
+
+variable "tfe_hostname" {
+  description = "The hostname to be used for reaching the deployed TFE instance"
+  type        = string
+}
+
+variable "tfe_tls_cert_mount_path" {
+  description = "The path where tls certificates will be mounted inside TFE"
+  type        = string
+  default     = "/etc/ssl/private/terraform-enterprise"
+}
+
+variable "tfe_iact_subnets" {
+  description = "The IACT subnet values to be used by TFE"
+  type        = string
+  default     = ""
+}
+
+variable "tfe_iact_time_limit" {
+  description = "The IACT time limit value to be used by TFE"
+  type        = string
+  default     = "60"
+}
+
+variable "tfe_resource_cpu" {
+  description = "The CPU resource limits to be used by TFE."
+  type        = number
+  default     = 750
+}
+
+variable "tfe_resource_memory" {
+  description = "The memory resource limits to be used by TFE."
+  type        = number
+  default     = 1024
+}
+
+variable "tfe_image" {
+  description = "The terraform enterprise image that will be used to deploy TFE"
+  type        = string
+  default     = "images.releases.hashicorp.com/hashicorp/terraform-enterprise:v202401-2"
```

Comment:
This is redundant with the server_address below, it should only be the namespace/image:version@sha, without the server address for easier consistency and to avoid repetition.
(so hashicorp/terraform-enterprise:v202401-2@sha which I don't know)